lower_prey: [1,125,255]
upper_prey: [100,255,255]
lower_predator: [133,15,250]
upper_predator: [153,210,255]
